### PR TITLE
Feat/live 2710 analytics asset wallet centric

### PR DESCRIPTION
**Pull request by ThomasLaforge**
### 📝 Description

Add Analytics to Asset Centric Wallet new content

### ❓ Context

- **Impacted projects**: `LLM` 
- **Linked resource(s)**: [Tracking Plan](https://ledgerhq.atlassian.net/wiki/spaces/WALLETCO/pages/3720806401/ASSET-CENTRIC-WALLET+-+Data+Tracking+Plan) 

### ✅ Checklist

- [ ] **Test coverage** 
- [x] **Atomic delivery** 
- [ ] **No breaking changes** 

### 🚀 Expectations to reach

_Please make sure you follow these [**Important Steps**](https://github.com/LedgerHQ/ledger-live/blob/develop/CONTRIBUTING.md#important-steps)._

_Pull Requests must pass the CI and be internally validated in order to be merged._

### CODE REVIEW

File: apps/ledger-live-mobile/src/components/Carousel/Slide.tsx [hunk 1]
```diff
@@ -39,16 +40,19 @@ const Slide = ({
 }: SlideProps) => {
   const { t } = useTranslation();
   const { navigate } = useNavigation();
+  const currentScreen = useCurrentRouteName();
   const onClick = useCallback(() => {
-    track("Portfolio Recommended OpenUrl", {
+    track("banner_clicked", {
+      banner: "Discover Web3",
```
Comment (cgrellard-ledger) [inline apps/ledger-live-mobile/src/components/Carousel/Slide.tsx:46]: I'm not sure about this one, why are we passing `Discover Web3` for all the slides ? Shouldn't we pass the title instead ?

File: apps/ledger-live-mobile/src/components/DiscreetModeButton.tsx [hunk 2]
```diff
@@ -4,13 +4,20 @@ import { useDispatch, useSelector } from "react-redux";
...
+import { useCurrentRouteName } from "../helpers/routeHooks";
+import { track } from "../analytics";
 
 export default function DiscreetModeButton({ size = 24 }: { size?: number }) {
   const discreetMode = useSelector(discreetModeSelector);
+  const currentScreen = useCurrentRouteName();
   const dispatch = useDispatch();
   const onPress = useCallback(() => {
+    track("button_clicked", {
+      button: "Mask",
```
Comment (cgrellard-ledger) [inline apps/ledger-live-mobile/src/components/DiscreetModeButton.tsx:16]: Is `Mask` the right event name even when we switch the discreetMode from true to false ?

File: apps/ledger-live-mobile/src/components/Carousel/Slide.tsx (hunk 1 above)
Comment (ThomasLaforge) [reply apps/ledger-live-mobile/src/components/Carousel/Slide.tsx:46]: image
But yes title of card clicked should be the one to go. I was ust using the spec here.

Comment (ThomasLaforge) [reply apps/ledger-live-mobile/src/components/Carousel/Slide.tsx]: changed as banner name

File: apps/ledger-live-mobile/src/components/DiscreetModeButton.tsx (hunk 2 above)
Comment (ThomasLaforge) [reply apps/ledger-live-mobile/src/components/DiscreetModeButton.tsx:16]: I send now Mask / Unmask depending on discreetMode value

**Review by cgrellard-ledger — APPROVED** [review]

**Pull request merged**